pkg/agent/variables.go

```diff
@@ -82,7 +82,19 @@ func getCSECommandVariables(config *datamodel.NodeBootstrappingConfiguration) pa
 		"auditdEnabled":                   strconv.FormatBool(to.Bool(profile.AuditDEnabled)),
 		"configGPUDriverIfNeeded":         config.ConfigGPUDriverIfNeeded,
 		"enableGPUDevicePluginIfNeeded":   config.EnableGPUDevicePluginIfNeeded,
+		"enableTelemetry":                 cs.Properties.FeatureFlags.IsFeatureEnabled("EnableTelemetry"),
+		"apiVersionNetwork":               datamodel.APIVersionNetwork,
 	}
+
+	if cs.Properties.HasWindows() {
+		cseCommandVariables["windowsEnableCSIProxy"] = cs.Properties.WindowsProfile.IsCSIProxyEnabled()
+		cseCommandVariables["windowsCSIProxyURL"] = cs.Properties.WindowsProfile.CSIProxyURL
+		cseCommandVariables["windowsProvisioningScriptsPackageURL"] = cs.Properties.WindowsProfile.ProvisioningScriptsPackageURL
+		cseCommandVariables["windowsPauseImageURL"] = cs.Properties.WindowsProfile.WindowsPauseImageURL
+		cseCommandVariables["alwaysPullWindowsPauseImage"] = strconv.FormatBool(cs.Properties.WindowsProfile.IsAlwaysPullWindowsPauseImage())
+	}
```

Comment:
I don't see we use these in cse command ? Instead they're used in getting node custom data

Reply:
This PR is for Windows CSE and Custom data, sorry for the misleading, I changed the PR title.

Reply:
Moved these into CustomData